django/contrib/auth/forms.py

```diff
@@ -253,8 +254,11 @@ def send_mail(self, subject_template_name, email_template_name,
         subject = loader.render_to_string(subject_template_name, context)
         # Email subject *must not* contain newlines
         subject = ''.join(subject.splitlines())
-        body = loader.render_to_string(email_template_name, context)
-
+        try:
+            body = loader.render_to_string(email_template_name, context)
+        except TemplateDoesNotExist:
```

Comment:
Since we would be shipping the default `.txt` template, it would never be missing, even if the user had previously overridden the old `.html` template, so this exception would never be raised, so their override would never be picked up.